java-checks/src/test/files/checks/ParameterReassignedToCheck.java

```diff
@@ -1,6 +1,19 @@
 import java.util.function.Supplier;
 
 class A {
+
+  public String foo(int dt) {
+    return switch (dt) {
+      case 1 -> {
+        try {
+          yield "";
+        } catch (Exception e) {
+          yield "Exc";
+        }
+      }
+      default -> null;
+    };
+  }
```

Comment:
This is a nice example to show that we do not break on switch expressions that yield. But it if feels gratuitous without a compliant/noncompliant comment in the sample for a specific rule. Can we replace this with/add a non-compliant example?